package/kernel/qca-nss-dp/Makefile

```diff
@@ -12,6 +12,11 @@ PKG_MIRROR_HASH:=45568d7f1268b67d752f7085f6ef8397ca8ee4e5456ef121b8a285bded99dc8
 PKG_BUILD_PARALLEL:=1
 PKG_FLAGS:=nonshared
 
+PKG_CONFIG_DEPENDS:= \
+	CONFIG_TARGET_qualcommax_ipq807x \
```

Comment:
Using just `CONFIG_TARGET_SUBTARGET` should do the same trick while not having to update the list.

Can you please try using that?